hirundo/cli.py

```diff
@@ -123,11 +152,37 @@ def setup(
     Setup the Hirundo client library.
     """
     api_host = fix_api_host(api_host)
-    upsert_env("API_HOST", api_host)
-    upsert_env("API_KEY", api_key)
-    print(
-        "API host and API key saved to .env for future use. Please do not share this file"
-    )
+    api_host_saved_to = upsert_env("API_HOST", api_host)
+    api_key_saved_to = upsert_env("API_KEY", api_key)
+    if api_host_saved_to != api_key_saved_to:
+        print(
+            "API host and API key saved to different locations. This should not happen. Please report this issue."
+        )
+        if api_host_saved_to == EnvLocation.HOME.name:
+            print(
+                "API host saved to ~/.hirundo.conf for future use. Please do not share the ~/.hirundo.conf file"
+            )
+        elif api_host_saved_to == EnvLocation.DOTENV.name:
+            print(
+                "API host saved to .env for future use. Please do not share this file"
+            )
+        if api_key_saved_to == EnvLocation.HOME.name:
+            print(
+                "API key saved to ~/.hirundo.conf for future use. Please do not share the ~/.hirundo.conf file since it contains your secret API key."
+            )
+        elif api_key_saved_to == EnvLocation.DOTENV.name:
+            print(
+                "API key saved to local .env file for future use. Please do not share the .env file since it contains your secret API key."
+            )
```

Comment:
```suggestion
        if api_host_saved_to == EnvLocation.HOME.name and api_key_saved_to == EnvLocation.DOTENV.name:
            print(
                "API host saved to ~/.hirundo.conf for future use. Please do not share the ~/.hirundo.conf file"
            )
            print(
                "API key saved to local .env file for future use. Please do not share the .env file since it contains your secret API key."
            )
        elif api_host_saved_to == EnvLocation.DOTENV.name and api_key_saved_to == EnvLocation.HOME.name:
            print(
                "API host saved to .env for future use. Please do not share this file"
            )
            print(
                "API key saved to ~/.hirundo.conf for future use. Please do not share the ~/.hirundo.conf file since it contains your secret API key."
            )
        else:
            # Should we throw an error here?
```

Reply:
The `else` can never happen since the other if checks if they are different and they are the only possible values for `api_host_saved_to` and `api_key_saved_to`